test/e2e_node/topology_manager_test.go

```diff
@@ -48,7 +49,10 @@ import (
 )
 
 const (
-	numalignCmd = `export CPULIST_ALLOWED=$( awk -F":\t*" '/Cpus_allowed_list/ { print $2 }' /proc/self/status); env; sleep 1d`
+	NUMAAlignmentCommand      = `export CPULIST_ALLOWED=$( awk -F":\t*" '/Cpus_allowed_list/ { print $2 }' /proc/self/status); env;`
+	NUMAAlignmentSleepCommand = NUMAAlignmentCommand + `sleep 1d;`
```

Comment:
do you need these identifers to be exported? please rename `numaAlignment...` if not

Reply:
Thank you for catching it, these are not needed to be exported - I renamed them.